packages/notifications/src/redux/action-types.js

```diff
@@ -2,3 +2,4 @@ const notificationsPrefix = '@@INSIGHTS-CORE/NOTIFICATIONS/';
 export const ADD_NOTIFICATION = `${notificationsPrefix}ADD_NOTIFICATION`;
 export const REMOVE_NOTIFICATION = `${notificationsPrefix}REMOVE_NOTIFICATION`;
 export const CLEAR_NOTIFICATIONS = `${notificationsPrefix}CLEAR_NOTIFICATIONS`;
+export const REMOVE_ENTITY = `${notificationsPrefix}REMOVE_ENTITY`;
```

Comment:
Wrong file and wrong type. It should be in `/packages/inventory/src/redux/action-types` in the list of `asyncInventory` as new array element.